<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsDockWidget</class>
 <widget class="QDockWidget" name="ToolsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="moveRadioButton">
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chooseColorRadioButton">
        <property name="text">
         <string>Choose Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="drawMaskRadioButton">
        <property name="text">
         <string>Draw Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="eraseMaskRadioButton">
        <property name="text">
         <string>Erase Mask</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Mask Brush Size</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QSpinBox" name="maskBrushSizeSpinBox"/>
         </item>
         <item>
          <widget class="QSlider" name="maskBrushSizeSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="completelyEraseMaskButton">
          <property name="text">
           <string>Clear Mask</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="completelyFillMaskButton">
          <property name="text">
           <string>Fill Mask</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Item Size Range</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QSpinBox" name="minSizeSpinBox"/>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="maxSizeSpinBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="maxLabel">
             <property name="text">
              <string>Max</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="minLabel">
             <property name="text">
              <string>Min</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="maxSizeSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSlider" name="minSizeSlider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
